index.js

```diff
@@ -0,0 +1,7 @@
+function addNumbers() {
+    var numberOne = document.getElementById("numberone").value
+    var numberTwo = document.getElementById("numbertwo").value
+
+    var results = parseInt(numberOne) + parseInt(numberTwo)
```

Comment:
what about numbers that are not ints... 

might need to use parseFloat() or Number()